javascript/builtins/Intl.json

```diff
@@ -708,14 +708,21 @@
                   "version_added": false
                 },
                 "chrome": {
```

Comment:
If there are multiple version ranges here, this needs to be an array instead of an object. Travis is failing because the JSON is invalid here.

Reply:
Ok. thank you. I'll fix that.